lib/certifi/version.rb

```diff
@@ -1,3 +1,4 @@
 module Certifi
-  VERSION = "2018.01.18"
+  VERSION_PATH = File.expand_path(File.join(File.dirname(__FILE__), "VERSION"))
+  VERSION      = File.read(VERSION_PATH)
```

Comment:
It seemed simplest to have the version stored in a separate file instead of trying to have the Rakefile rewrite this Ruby file.